[1.6] Empty uC crash

When someone on my server placed 'empty' uC from NEI everyone who have it in their view distance is getting following crashes:
```
Caused by: java.lang.NullPointerException
        at li.cil.oc.common.tileentity.Microcontroller.func_70302_i_(Microcontroller.scala:232)
        at li.cil.oc.common.tileentity.traits.ComponentInventory$class.$init$(ComponentInventory.scala:24)
        at li.cil.oc.common.tileentity.Microcontroller.<init>(Microcontroller.scala:25)
        at li.cil.oc.common.block.Microcontroller.createTileEntity(Microcontroller.scala:68)
        at li.cil.oc.common.block.Microcontroller.createTileEntity(Microcontroller.scala:26)
        at net.minecraft.world.chunk.Chunk.func_150806_e(Chunk.java:850)
        at net.minecraft.world.World.func_147438_o(World.java:2540)
        at li.cil.oc.common.block.RedstoneAware.func_149695_a(RedstoneAware.scala:53)
        at net.minecraft.world.World.func_147460_e(World.java:697)
```

--- a/src/main/scala/li/cil/oc/common/tileentity/traits/ComponentInventory.scala
+++ b/src/main/scala/li/cil/oc/common/tileentity/traits/ComponentInventory.scala
@@ -21,8 +21,8 @@ trait ComponentInventory extends Environment with Inventory with inventory.Compo
   // Cache changes to inventory slots on the client side to avoid recreating
   // components when we don't have to and the slots are just cleared by MC
   // temporarily.
-  private val pendingRemovalsActual = mutable.ArrayBuffer.fill(getSizeInventory)(None: Option[ItemStack])
-  private val pendingAddsActual = mutable.ArrayBuffer.fill(getSizeInventory)(None: Option[ItemStack])
+  private lazy val pendingRemovalsActual = mutable.ArrayBuffer.fill(getSizeInventory)(None: Option[ItemStack])
+  private lazy val pendingAddsActual = mutable.ArrayBuffer.fill(getSizeInventory)(None: Option[ItemStack])
   private var updateScheduled = false
   def pendingRemovals = {
     adjustSize(pendingRemovalsActual)

